app/javascript/pages/admin/Events/new.js

```diff
@@ -15,11 +15,32 @@ import CreateEventMutation from './mutations/create.gql'
 
 import Loading from 'components/LoadingIcon'
 
-const NewEvent = ({ createEvent, data: { networkStatus, eventTypes, offices, organizations } }) =>
+const transformToReduxFormState = event => {
```

Comment:
A note with whether to use graphQL to get the below shape for the redux-form state:
- I don't think it's worth it, as you'd need to create another graphQL query (duplicate query, and extra latency) to get the same information the existing query captures (existing query gets all the necessary information, and more - for form options)
- You'll still need to create a function to parse the dates retrieved from graphQL to a javascript `Date` object in which the redux-form state is configured.

This solution uses the existing event information retrieved and transforms it to the shape that redux-form is configured with.